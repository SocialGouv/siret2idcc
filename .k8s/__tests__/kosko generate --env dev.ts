//

import { getEnvManifests } from "@socialgouv/kosko-charts/testing";
import { project } from "@socialgouv/kosko-charts/testing/fake/gitlab-ci.env";

jest.setTimeout(1000 * 60);
test("kosko generate --dev", async () => {
  expect(
    await getEnvManifests("dev", "", project("siret2idcc").dev)
  ).toMatchSnapshot();
});
