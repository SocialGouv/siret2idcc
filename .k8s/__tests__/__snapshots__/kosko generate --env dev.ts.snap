// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kosko generate --dev 1`] = `
"---
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    field.cattle.io/creatorId: gitlab
    field.cattle.io/projectId: c-bd7z2:p-dtlsm
    git/branch: master
    git/remote: >-
      https://gitlab-ci-token:[MASKED]@gitlab.factory.social.gouv.fr/SocialGouv/siret2idcc.git
    app.gitlab.com/app: socialgouv-siret2idcc
    app.gitlab.com/env: master-dev2
    app.gitlab.com/env.name: master-dev2
  labels:
    azure-pg-admin-user: siret2idcc
    application: master-dev2-siret2idcc
    owner: siret2idcc
    team: siret2idcc
    cert: wildcard
  name: siret2idcc-65-master-dev2
---
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-siret2idcc
    app.gitlab.com/env: master-dev2
    app.gitlab.com/env.name: master-dev2
  labels:
    app: siret2idcc
    application: master-dev2-siret2idcc
    owner: siret2idcc
    team: siret2idcc
    cert: wildcard
  name: siret2idcc
  namespace: siret2idcc-65-master-dev2
spec:
  replicas: 1
  selector:
    matchLabels:
      app: siret2idcc
  template:
    metadata:
      annotations:
        kapp.k14s.io/disable-default-ownership-label-rules: ''
        kapp.k14s.io/disable-default-label-scoping-rules: ''
        app.gitlab.com/app: socialgouv-siret2idcc
        app.gitlab.com/env: master-dev2
        app.gitlab.com/env.name: master-dev2
      labels:
        app: siret2idcc
        application: master-dev2-siret2idcc
        owner: siret2idcc
        team: siret2idcc
        cert: wildcard
    spec:
      containers:
        - image: >-
            harbor.fabrique.social.gouv.fr/cdtn/siret2idcc:0123456789abcdefghijklmnopqrstuvwxyz0123
          livenessProbe:
            failureThreshold: 6
            httpGet:
              path: /healthz
              port: http
            initialDelaySeconds: 30
            periodSeconds: 5
            timeoutSeconds: 5
          name: siret2idcc
          ports:
            - containerPort: 3000
              name: http
          readinessProbe:
            failureThreshold: 15
            httpGet:
              path: /healthz
              port: http
            initialDelaySeconds: 0
            periodSeconds: 5
            successThreshold: 1
            timeoutSeconds: 1
          resources:
            limits:
              cpu: 500m
              memory: 1280Mi
            requests:
              cpu: 100m
              memory: 128Mi
          startupProbe:
            failureThreshold: 12
            httpGet:
              path: /healthz
              port: http
            periodSeconds: 5
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: siret2idcc
    application: master-dev2-siret2idcc
    owner: siret2idcc
    team: siret2idcc
    cert: wildcard
  name: siret2idcc
  annotations:
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-siret2idcc
    app.gitlab.com/env: master-dev2
    app.gitlab.com/env.name: master-dev2
  namespace: siret2idcc-65-master-dev2
spec:
  ports:
    - name: http
      port: 80
      targetPort: 3000
  selector:
    app: siret2idcc
  type: ClusterIP
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    kapp.k14s.io/disable-default-ownership-label-rules: ''
    kapp.k14s.io/disable-default-label-scoping-rules: ''
    app.gitlab.com/app: socialgouv-siret2idcc
    app.gitlab.com/env: master-dev2
    app.gitlab.com/env.name: master-dev2
  labels:
    app: siret2idcc
    application: master-dev2-siret2idcc
    owner: siret2idcc
    team: siret2idcc
    cert: wildcard
  name: siret2idcc
  namespace: siret2idcc-65-master-dev2
spec:
  rules:
    - host: master-dev2-siret2idcc.dev2.fabrique.social.gouv.fr
      http:
        paths:
          - backend:
              serviceName: siret2idcc
              servicePort: 80
            path: /
  tls:
    - hosts:
        - master-dev2-siret2idcc.dev2.fabrique.social.gouv.fr
      secretName: wildcard-crt
"
`;
